site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.test.ts

```diff
@@ -155,3 +158,45 @@ describe("validationSchema", () => {
     expect(validate).toThrowError("ttl must be less than or equal to 168")
   })
 })
+
+describe("ttlShutdownAt", () => {
+  it.each<[dayjs.Dayjs, Workspace, string, number, string]>([
+    [
+      dayjs("2022-05-17T18:09:00Z"),
+      Mocks.MockStoppedWorkspace,
+      "America/Chicago",
+      1,
+      "Your workspace will automatically shut down after this amount of time has elapsed.",
```

Comment:
**Suggestion(blocking)**: Use the `Language` object for these instead of duping them